backend/typescript/rest/campRoutes.ts

```diff
@@ -33,6 +33,16 @@ campRouter.get("/", async (req, res) => {
   }
 });
 
+campRouter.get("/:id", async (req, res) => {
+  try {
+    const camp = await campService.getCampById(req.params.id);
+
+    res.status(200).json(camp);
+  } catch (error: unknown) {
+    res.status(500).json({ error: getErrorMessage(error) });
```

Comment:
If the user sends a get request with a wrong camp id, it should return a 400 series error, not 500. recommend adding an if statement or something to verify which error is being sent back.

Reply:
This can also be done using some kind of validator middleware maybe?

Reply:
I think this is fine to leave as is! returning the error message is good enough imo, we can display the message in the frontend